dotnet/src/SemanticKernel.Skills/Skills.Grpc/Skills.Grpc.csproj

```diff
@@ -0,0 +1,29 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  
+  <PropertyGroup>
+    <RepoRoot>$([System.IO.Path]::GetDirectoryName($([MSBuild]::GetPathOfFileAbove('.gitignore', '$(MSBuildThisFileDirectory)'))))</RepoRoot>
+  </PropertyGroup>
+  <Import Project="$(RepoRoot)/dotnet/nuget/nuget-package.props" />
+
+  <PropertyGroup>
+    <AssemblyName>Microsoft.SemanticKernel.Skills.Grpc</AssemblyName>
+    <RootNamespace>Microsoft.SemanticKernel.Skills.Grpc</RootNamespace>
+    <TargetFramework>netstandard2.1</TargetFramework>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <!-- NuGet Package Settings -->
+    <PackageId>Microsoft.SemanticKernel.Skills.Grpc</PackageId>
+    <Title>Semantic Kernel - gRPC Skill</Title>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Google.Protobuf" Version="3.22.1" />
```

Comment:
Please add the packages to the Directory.Packages.props file and remove the version attributes from the PackageReference attributes. It will fix the compilation issue - _Error	NU1008	Projects that use central package version management should not define the version on the PackageReference items but on the PackageVersion items....
Same for the new packages added to Skills.UnitTests project.

Reply:
Moved it to `Directory.Packages.props` and it builds successfully on windows:
![image](https://user-images.githubusercontent.com/18152044/232110290-e488f6e2-ae41-46bd-acc2-6972ff09b291.png)